src/odb/src/db/dbJournal.cpp

```diff
@@ -1544,18 +1544,21 @@ void dbJournal::undo_deleteObject()
       int y_max;
       uint layer_id;
       uint via_layer_id;
+      bool is_congested;
       _log.pop(net_id);
       _log.pop(x_min);
       _log.pop(y_min);
       _log.pop(x_max);
       _log.pop(y_max);
       _log.pop(layer_id);
       _log.pop(via_layer_id);
+      _log.pop(is_congested);
```

Comment:
I don't see this being pushed in the destroy method

Reply:
Where exactly it should be pushed? I was following the same logic I did with via_layer_id and I'm not sure where whould I do that.

Reply:
dbGuide::destroy

Reply:
You did add 
```
    block->_journal->pushParam(_guide->via_layer_);
```
previously.

Reply:
Thanks! Fixed it now.